frontend/src/styles/modules/_buttons.scss

```diff
@@ -145,6 +145,10 @@
     background-color: $disabled-grey;
   }
 
+  &.big-border {
```

Comment:
> we could have a standalone sass page for variants that are namespaced as such like `class=variant--big-border`

This is the right answer. (Well, CSS modules are, but this is a solid runner-up)